Solutions/Valence Security/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "...",
```

Comment:
Hello @shlomimatichin, Please add publisherId. It cannot be kept empty

Reply:
i see a lot of solutions use "azuresentinel", can i use that?

Reply:
found it, done.